mod-linked-data/src/main/resources/citation/mod-linked-data/features/create-bib-api/create-bib-api.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Create Work and Instance resource using API & validate that they is created in mod-inventory and mod-search
```

Comment:
minor
...validate that they **are** created...
same on line 9

Reply:
Done